OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ChatWorkshopController.cs

```diff
@@ -167,7 +171,7 @@ private async Task<bool> IsProviderAChatRoomParticipantAsync(ChatRoomWorkshopDto
         {
             var userId = this.GetUserId();
 
-            var result = await validationService.UserIsWorkshopOwnerAsync(userId, chatRoom.WorkshopId).ConfigureAwait(false);
+            var result = await validationService.UserIsWorkshopOwnerAsync(userId, chatRoom.WorkshopId, GetUserSubrole()).ConfigureAwait(false);
```

Comment:
please don't mix passing variable and method invocation.
you already using `var userId = this.GetUserId();`, so why don't the same with `GetUserSubrole` :) especially if you're doing just that in another method

Reply:
Done